content/collections/agents/en/ai-feedback.md

```diff
@@ -0,0 +1,171 @@
+---
+id: 61027aa7-267f-479f-b379-912c1fea207f
+blueprint: agent
+title: 'AI Feedback'
+this_article_will_help_you:
+  - 'Understand what AI Feedback is'
+  - 'Link your data sources to AI Feedback'
+  - 'Create actions based on the insights provided by AI Feedback'
+updated_by: 3f7c2286-b7da-4443-a04f-7c225af40254
+updated_at: 1761239369
+---
+AI Feedback is an AI-native tool that combines customer voice, product behavior and AI-powered insights into actionable workflows. AI Feedback monitors feedback sources that you define and then delivers actionable insights through Amplitude. AI Feedback groups customer feedback into categories as well as connecting feedback insights into Amplitude cohorts, session replays and experiments. AI Feedback also lets you create guides and surveys based off of the insights to better connect you with your customers’ needs. 
```

Comment:
```suggestion
AI Feedback is an AI-native tool that combines customer voice, product behavior, and AI-powered insights into actionable workflows. AI Feedback monitors feedback sources that you define and delivers actionable insights through Amplitude. AI Feedback groups customer feedback into categories as well as connecting feedback insights into Amplitude cohorts, session replays and experiments. AI Feedback also lets you create guides and surveys based off of the insights to better connect you with your customers’ needs. 
```
Each sentence in this paragraph starts with `AI Feedback`. Is it worth rewriting a bit?